core/media_streamer_Internal_test.go

```diff
@@ -20,7 +20,8 @@ var _ = Describe("MediaStreamer", func() {
 
 	BeforeEach(func() {
 		DeferCleanup(configtest.SetupConfig())
-		conf.Server.DataFolder, _ = os.MkdirTemp("", "file_caches")
+		conf.Server.DataFolder, _ = os.MkdirTemp("", "data")
+		conf.Server.CacheFolder, _ = os.MkdirTemp("", "file_caches")
```

Comment:
Tests are passing without adding this line.

Reply:
Removed those as suggested